tests/e2e-pw/types.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Interface that describes the properties of a block, used to help with finding/editing the block in E2E tests.
+ */
+export interface BlockTestingProperties {
+	title: string;
+	selectors: Record< string, string >;
+	slug: string;
+}
```

Comment:
While I'm not an expert in TS, I wonder if we should change
```ts
export interface BlockTestingProperties {
	title: string;
	selectors: Record< string, string >;
	slug: string;
}
```
to
```ts
export interface BlockTestingProperties {
	title: string;
	selectors: {
		className: string;
		insertButton: string;
	};
	slug: string;
}
```
Which the first approach `selectors: Record< string, string >;`, `selectors` can hold any data, couldn't it? I mention this based on my earlier comment of changing
```tsx
const blockProperties: BlockTestingProperties = {
	title: 'Cart',
	slug: 'woocommerce/cart',
	selectors: {
		block: '.wp-block-woocommerce-cart',
		insertButton: "//button//span[text()='Cart']",
	},
};
```
to
```tsx
const blockProperties: BlockTestingProperties = {
	title: 'Cart',
	slug: 'woocommerce/cart',
	selectors: {
		className: '.wp-block-woocommerce-cart',
		insertButton: "//button//span[text()='Cart']",
	},
};
```

With a more strict typing, we'd catch that immediately. With a more generic typing, we would not catch this. That said, I'm not an expert in TS and I might be mistaken in this case.